code/datums/gamemode/role/ninja.dm

```diff
@@ -17,7 +17,8 @@
 	. =..()
 	if(!.)
 		return
-	antag.current.forceMove(pick(ninjastart))
+	spawn(3) //to ensure other running processes don't throw them somewhere else
```

Comment:
this is horrible

Reply:
even worse is that it's probably unnecessary